src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py

```diff
@@ -318,15 +323,17 @@ def acr_queue(cmd,
     else:
         print("Sending build context to ACR")
 
-    result = LongRunningOperation(cmd.cli_ctx)(client_registries.queue_build(
+    queued_build = LongRunningOperation(cmd.cli_ctx)(client_registries.queue_build(
         build_request=build_request, resource_group_name=resource_group_name,
         registry_name=registry_name))
 
-    print("Queued a build with ID: {0}".format(result.build_id))
-
-    if no_logs == False:
-        acr_build_show_logs(cmd, client, registry_name,
-                            result.build_id, resource_group_name)
+    if no_logs:
+        return queued_build
```

Comment:
What is printed here? This seems to be the result of a template deployment that wouldn't be much useful. In `acr create` we do a `GET` after template deployment to ensure the correct information is shown.

Reply:
I don't think this return value is used, I would leave the code as is. I still want the build id after executing a quick build even, otherwise there's no way to tell what number I queued up is from the terminal.

Reply:
this is useful when user specifies:
 --no-logs -o json